modules/markdown/easy-markdown.php

```diff
@@ -701,15 +701,15 @@ public function xmlrpc_actions( $xmlrpc_method ) {
 	 * @return null
 	 */
 	protected function check_for_early_methods() {
-		global $HTTP_RAW_POST_DATA;
-		if ( false === strpos( $HTTP_RAW_POST_DATA, 'metaWeblog.getPost' )
-			&& false === strpos( $HTTP_RAW_POST_DATA, 'wp.getPage' ) ) {
+		$raw_post_data = file_get_contents( "php://input" );
```

Comment:
Just noting that we're good here. Tested this with PHP 5.4, and it doesn't interfere with XMLRPC requests because:
* XMLRPC request the input stream first, only passing control here later,
* The manual says that mostly POST requests should be good to restart reading from input even for older versions because POST requests are saved.
* This code is only supposed to work in metaWeblog and generic WordPress XMLRPC requests in conjunction with Markdown.
All this to say that we're OK.